woocommerce/woocommerce
# Move remaining CES-related components to @woocommerce/customer-effort-score

### All Submissions:






### Changes proposed in this Pull Request:



This PR moves the remaining CES-related components currently found in `woocommerce/admin/client` to the `@woocommerce/customer-effort-score` package. This builds on the work in #37252 and #37112 and is necessary for #36722.

**No functionality should be impacted by this change... this is just a package file refactoring.**

Part of #36722.



### How to test the changes in this Pull Request:




Note: make sure tracking is enabled under WooCommerce > Settings > Advanced > WooCommerce.com

In order for CES snackbars and such to show up, you need to have not already dismissed/completed the CES for a particular feature. To reset things, you can delete the `woocommerce_ces_shown_for_actions` transient (use the WCA Test Helper).

1. Load this branch and run `pnpm install && pnpm run build` and make sure everything builds correctly.
2. Smoke test WCAdmin pages and make sure that any CES-related functionality works correctly.
   
    * Import CES snackbar - Go to Products > All Products; click "Import"; without continuing, click back to Products > All Products (see testing instructions on #36656)
    * Filtering snackbar - Use advanced filter in Analytics > Variations; enter advanced filter criteria and click "Filter"
    * CES footer on new product editor page - make sure "Share feedback" button brings up feedback modal.

3. Open your console and check if the following exist under `window.wc.customerEffortScore` (there will be more, these are the new ones added in this PR):

- `ADMIN_INSTALL_TIMESTAMP_OPTION_NAME`
- `CustomerEffortScore`
- `CustomerEffortScoreModalContainer`
- `CustomerEffortScoreTracksContainer`
- `CustomerEffortScoreTracks`
- `getStoreAgeInWeeks`
- `SHOWN_FOR_ACTIONS_OPTION_NAME`



### Other information:




### FOR PR REVIEWER ONLY:
